Eat_and_Greet/src/screens/HomeScreen/index.jsx

```diff
@@ -0,0 +1,28 @@
+import { StyleSheet, Text, View } from 'react-native';
+import { StatusBar } from 'expo-status-bar';
+
+import { Toolbar } from '../../../comps/Toolbar/toolbar';
+import TextBox from '../../../comps/Toolbar/textbox';
+
+function HomeScreen() {
+    return (
+      <View style={styles.container}>
+        <Toolbar />
+        <StatusBar style="auto" />
+        <TextBox placeholder="Type ..." />
+        <Text>Home Screen Bruh</Text>
+        <Text>Welcome to our app!</Text>
+      </View>
+    );
+}
+
+export default HomeScreen;
+
+const styles = StyleSheet.create({
```

Comment:
This container styling will probably need adjusting later, I would suggest putting the container in the general style page for consistency in the program.

Reply:
Not blocking the pull request.